tests/macros/texpectIdent2.nim

```diff
@@ -0,0 +1,24 @@
+discard """
```

Comment:
these failure files are not megatest-joinable and add to CI test time; instead you can merge files with expected failures using `nim check`, see example here: tests/errmsgs/tsigmatch.nim

Reply:
Actually, it looks like this wouldn't work, somehow `error` aborts on 1st error, even with `nim check`, and even with `--errormax:10`; not sure why; so maybe ignore this (but seems like a bug in `nim check`)